CodeGeneration/Sources/SyntaxSupport/ExprNodes.swift

```diff
@@ -1497,6 +1497,38 @@ public let EXPR_NODES: [Node] = [
     elementChoices: [.stringSegment, .expressionSegment]
   ),
 
+  Node(
+    kind: .simpleStringLiteralExpr,
+    base: .expr,
+    nameForDiagnostics: "simple string literal",
+    documentation: "A simple string that can't be used for string interpolation or extended escaping.",
```

Comment:
Also tried to add some documentation. Improvements are welcome!

Reply:
Thank you. That’s very appreciated.

I would go for

```suggestion
    documentation: "A simple string that can’t contain string interpolation and cannot have raw string delimiters.",
```